src/ucp/tag/rndv.c

```diff
@@ -716,7 +725,8 @@ UCS_PROFILE_FUNC_VOID(ucp_rndv_matched, (worker, rreq, rndv_rts_hdr),
         }
         /* put protocol is allowed - register receive buffer memory for rma */
         ucp_request_recv_buffer_reg(rreq, ucp_ep_config(ep)->key.rma_bw_md_map,
-                                    ucs_min(rreq->recv.length, rndv_rts_hdr->size));
+                                    ucs_min(rreq->recv.length,
```

Comment:
this change does not seem needed as well, rreq->recv.tag.info.length and rndv_rts_hdr->size are the same

Reply:
can't they be different when receiver posts bigger than actual data buffer?

Reply:
line 669 sets req->recv.tag.info.length to rndv_rts_hdr->size, i don't see anything else changes it..

Reply:
> line 669 sets req->recv.tag.info.length to rndv_rts_hdr->size, i don't see anything else changes it..

I just wanted to align it with the place where we assign value to RTR length to make it consistent

Reply:
pls revert, "info" is used to return sender len to user, it should not be used for anything else

Reply:
right, thought it was just `rreq->recv.length` before

Reply:
done